### DISCUSSION TITLE
[Feature] Hardware-Accelerated Machine support edge tpu?

### DISCUSSION THREAD

**Discussion by ZhaiSoul** [immich-app/immich#13400] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a Coral M.2 Accelerator. I hope support for Hardware-Accelerated Machine Learning.
This may enable Raspberry Pi to also perform machine learning

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mertalev** [comment]
We looked into this some time ago, but the Coral is just too limited for the models we use. Hailo modules (used in the Raspberry Pi AI Kit) may be supported in the future, though.

**Reply by yayaasd** [reply]
Any news on this?

**Reply by bo0tzz** [reply]
The reason given is pretty definitive, what news would you expect? Also please read https://justinmayer.com/posts/any-updates/

**Reply by yayaasd** [reply]
Sorry was more asking for Hailo Modules - was not clear.
And yeah, thank you also for the link. I do understand if people asking ever 3 weeks, but getting an update about the idea after 3/4 year may be not punished ;-)

will follow and donate, thank you for your comment

**Reply by bo0tzz** [reply]
For Hailo modules, see  #17230
